src/System.Runtime.Serialization.Formatters/tests/FormatterServicesTests.cs

```diff
@@ -29,21 +34,260 @@ public void GetSerializableMembers_Interface()
         }
 
         [Fact]
-        public void GetUninitializedObject_InvalidArguments_ThrowsException()
+        public void GetUninitializedObject_NullType_ThrowsArgumentNullException()
         {
             Assert.Throws<ArgumentNullException>("type", () => FormatterServices.GetUninitializedObject(null));
             Assert.Throws<ArgumentNullException>("type", () => FormatterServices.GetSafeUninitializedObject(null));
         }
 
+#if netcoreapp
```

Comment:
You can create a non-runtime type by creating your local NonRumtimeType that inherits from Type. It will work everywhere. The netcoreapp ifdef would not be needed.